2018/README.md

```diff
@@ -0,0 +1,111 @@
+# Notes for 2018
+
+The templates here are intended to be 1:1 scale diagram of all the areas used in SUSU, they are believed to be within 5% accurate.
+Locations highlighted in red on one of the venue related layers require measurements to be taken to assure their size.
+
+#### Requires:
+- \> python 3.4
+- PyYaml
+
+## Layers
+#### Floorplan layers:
+- Notes *(Additional notes)*
+- Dimensions
+  - VenueDimensions *(Dimensions relating to venue floorplan)*
+  - AreaDimensions *(Dimensions relating to area placement and layout)*
+- TLA *(Team names)*
+- Network
+  - Displays *(SRcomp info monitors)*
+  - VenueNetwork *(Venue installed networking)*
+  - NetworkEquipment *(Additional cables, switches, routers and access points)*
+- Power
+  - VenuePower *(Venue power outlets)*
+  - PowerEquipment *(Required power locations)*
+- Areas
+  - AreaTables *(Position of designated tables)*
+  - AreaZones *(Marked out zones)*
+- Shepherding *(Shepherd specific highlighting)*
+- General
+  - GeneralLabels *(global drawing labels)*
+  - VenueFloorplan *(venue floorplans)*
+  - Exits *(Fire exits and lanes)*
+
+
+#### Key Layers:
+- General
+- Network
+- Power
+- Areas
+- Shepherding
+
+
+## Generation Script:
+#### Folder structure:
+```
+team_names.txt
+templates/
+  Lv4.svg
+  Lv3.svg
+  Lv2.svg
+  Cube.svg
+  key.svg
+  layout.svg
```

Comment:
`layout.svg` seems to be missing and the other names don't seem to match the names of the files in the `templates` directory. Is this expected?
This seems to be an issue as the `venue_map.yaml` file expects the `layout.svg` template to exist.

(meta: I'd be tempted to drop this sort of detail from the REAMDE, including specific file names here is a recipe for them getting out of date).